experimental/packages/opentelemetry-instrumentation-grpc/src/grpc-js/index.ts

```diff
@@ -334,4 +349,19 @@ export class GrpcJsInstrumentation extends InstrumentationBase {
       }
     });
   }
+
+  private _createMetadataCapture() {
+    const config = this.getConfig();
+
+    return {
+      client: {
+        captureRequestMetadata: metadataCapture('request', config.metadataToSpanAttributes?.client?.requestMetadata ?? []),
+        captureResponseMetadata: metadataCapture('response', config.metadataToSpanAttributes?.client?.responseMetadata ?? [])
+      },
+      server: {
+        captureRequestMetadata: metadataCapture('request', config.metadataToSpanAttributes?.server?.requestMetadata ?? []),
+        captureResponseMetadata: metadataCapture('response', config.metadataToSpanAttributes?.server?.responseMetadata ?? [])
+      },
+    };
+  }
```

Comment:
why is this transformation needed?

Reply:
to make sure no unexpected errors occur
I initialize with empty arrays if the configuration doesn't contain any metadata